packages/venia-ui/lib/components/CartPage/ProductListing/quantity.css

```diff
@@ -0,0 +1,69 @@
+.root {
+    text-align: center;
+    display: grid;
+    grid-template-columns: 1fr 4rem 1fr;
+    grid-column-gap: 0.5rem;
+    align-items: center;
+    justify-items: center;
+}
+
+.wrap {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.label:not(:focus):not(:active) {
+    position: absolute;
+    overflow: hidden;
+    width: 1px;
+    height: 1px;
+    clip: rect(0 0 0 0);
+    clip-path: inset(100%);
+    white-space: nowrap;
+}
+
+.button {
+    --size: 2.5rem;
```

Comment:
Leaving aside the mockups for a second, shouldn't this value match the input? (`2.25rem`) It looks off to me, as-is.

Reply:
Fair. I'll make that change in good faith. Will still need UX to sign off on this iteration.